File: spring/spring-comments/src/main/java/com/hcsy/spring/entity/po/Comments.java
package com.hcsy.spring.entity.po;

import java.time.LocalDateTime;

import com.baomidou.mybatisplus.annotation.TableId;
import com.baomidou.mybatisplus.annotation.TableName;

import lombok.Data;

@Data
@TableName("comments") // 对应数据库中的表名
public class Comments {
    @TableId
    private Long id;
    private String content;
    private Double star; // 星级评分，1~10
    private Long articleId;
    private Long userId;
    private LocalDateTime createTime;
    private LocalDateTime updateTime;
}

File: fastapi/entity/po/comments.py
from sqlmodel import SQLModel, Field
from typing import Optional

class Comments(SQLModel, table=True):
    """评论实体类"""
    
    __tablename__ = "comments"
    id: Optional[int] = Field(default=None, primary_key=True, index=True)
    content: str = Field()
    star: float = Field()
    user_id: int = Field()
    article_id: int = Field()
    create_time: Optional[str] = Field(default=None)
    update_time: Optional[str] = Field(default=None)